recipe/skip_problematic_test.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/sklearn/tests/test_random_projection.py b/sklearn/tests/test_random_projection.py
+index d23da948e..b87209195 100644
+--- a/sklearn/tests/test_random_projection.py
++++ b/sklearn/tests/test_random_projection.py
+@@ -116,7 +116,7 @@ def check_input_with_sparse_random_matrix(random_matrix):
+ 
+ 
+ @pytest.mark.parametrize("random_matrix", all_random_matrix)
+-def test_basic_property_of_random_matrix(random_matrix):
```

Comment:
why are we applying a different approach here?

Reply:
Because this test failed at collection time due to things being sorted differently.

Reply:
> Because this test failed at collection time due to things being sorted differently.

Right. we should have addressed order indeterminacy in https://github.com/scikit-learn/scikit-learn/commit/bec83089f7b6a27fd0a42d28305167ecea393c6a, included in v0.22